schemas/questionnaire_v1.json

```
@@ -176,6 +176,10 @@
             "type": "boolean",
             "description": "On completion of the section a summary will be shown"
           },
+          "show_on_hub": {
+            "type": "boolean",
+            "description": "When the hub is enabled, this will determine if the section appears. Default is True"
```

```suggestion
            "description": "When the hub is enabled, this will determine if the section appears. Default is true"
```

All other booleans referenced use JSON lowercase style